src/main/java/forestry/apiculture/genetics/Bee.java

```diff
@@ -376,6 +378,36 @@ public ArrayList<BiomeGenBase> getSuitableBiomes() {
         return suitableBiomes;
     }
 
+    @Override
+    public void addLore(List<String> text) {
+        IAlleleBeeSpecies primary = genome.getPrimary();
+        String descTokens[] = primary.getDescription().split("\\|");
+
+        if (descTokens.length > 0) {
+            String speciesLore = descTokens[0];
+            if (!speciesLore.isEmpty() && !speciesLore.contains("for.description") && Proxies.common.isCtrlDown()) {
+                FontRenderer fontRenderer = Proxies.common.getClientInstance().fontRenderer;
```

Comment:
You don't need to check for the control key here, because you do that in `ItemGE:addInformation`. Also why do you compare description with "for.description"?
```suggestion
            if (!speciesLore.isEmpty()) {
                FontRenderer fontRenderer = Minecraft.getMinecraft().fontRenderer;
```

Reply:
This check is also unnecessary for other implementations of this method.

Reply:
If the lang file doesn't provide a translated string for the localization, then the token for the name seems to default to "for.description". See here for the butterfly example: for.description.lepiCabbageWhite.

![image](https://github.com/user-attachments/assets/7685ce15-1162-44ce-bbc3-39ebff810fcc)